apps/app-article/src/components/article-details.js

```diff
@@ -61,7 +61,7 @@ const ArticleDetails = (props) => {
                                 }
                                 {
                                     props.premium ?
-                                        <PremiumBadge/>
+                                        <div className="article-info"><span className="article-opinion-type">{props.articleTypeDetails.title}</span> <PremiumBadge/></div>
```

Comment:
`articleTypeDetails` is not always returned from the API, so we need to check that it exists whenever we access its attributes.